static/styles/portico/portico.scss

```diff
@@ -168,30 +168,25 @@ html {
     line-height: 1.5;
     margin-bottom: 0px;
     border-bottom: 1px solid hsla(0, 0, 100%, .6);
-}
-
-.help .sidebar h1:not(:first-of-type) {
-    margin-top: 20px;
-}
-
-.help .sidebar h1:first-of-type,
-.help .sidebar h1.home-link {
-    border-bottom: none;
-}
 
+    &:not(:first-of-type) {
+        margin-top: 20px;
+    }
 
-.help .sidebar h1.home-link {
-    font-size: 1em;
-    margin-bottom: 17px;
-}
+    &:first-of-type,
+    &.home-link {
+        border-bottom: none;
+    }
 
-.help .sidebar h1.home-link a::before {
-    display: inline-block;
-    font: normal normal normal 14px/1 FontAwesome;
-    font-size: inherit;
+    &.home-link {
+        font-size: 1em;
+        margin-bottom: 17px;
+    }
 
-    content: "\f0d9";
```

Comment:
It looks like you lost this line in this translation; I think you may have lost the whole block.  

Please review your commits with `git show` using these `.gitconfig` options with modern Git to check for mistakes like this:
```
[diff]
        colorMovedWS = allow-indentation-change
        colorMoved = default
```

This SCSS nesting migration is only worth doing if we use a process that avoids any accidental changes to CSS while doing it, since it's very expensive to manually QA every page for whether the CSS looks wrong.  Please check carefully to make sure you don't have any similar issues in the rest of this branch.

Reply:
My bad, I will fix those